docs/resources.rst

```diff
@@ -144,6 +144,3 @@ Third-Party Solidity Parsers and Grammars
 
 * `solidity-parser <https://github.com/ConsenSys/solidity-parser>`_
```

Comment:
Actually this repository states:
> ⚠️ DEPRECATION NOTICE ⚠️
> This library is not being maintained anymore. For an up-to-date Solidity parser, check out [solidity-parser-antlr](https://github.com/federicobond/solidity-parser-antlr).

Reply:
Replaced with `solidity-parser-antlr`. But maybe I should remove it altogether? `solidity-parser-antlr` is just a JS lib that uses `solidityj/solidity-antlr4`.

Reply:
Yes but it is a tool, what is more problematic it is outdated. Opened an issue on it, but we'll see.

Reply:
I see no answer in the issue so far (https://github.com/federicobond/solidity-parser-antlr/issues/106) so I'm going to leave the link in. We can remove it later if it turns out that the project is dead or unusable.

Reply:
In the end I changed it to https://github.com/solidity-parser/parser mentioned by @franzihei in https://github.com/ethereum/solidity/issues/9369#issuecomment-658663373. It's a fork that is using the grammar from our repo and seems to be actively maintained.